pages/dkp/kommander/1.4/clusters/attach-cluster/requirements-for-attaching/index.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: layout.pug
+navigationTitle: Requirements for Attaching an Existing Cluster to Kommander
+title: Requirements for Attaching an Existing Cluster to Kommander
+menuWeight: 3
+excerpt: Requirements for attaching an existing cluster to Kommander
+beta: true
+---
+
+## Basic requirements
+
+To attach an existing cluster to Kommander, the Kommander management cluster must be able to reach the services and the api-server of the target cluster.
+
+You can attach existing clusters with or without networking restrictions to Kommander. These networking restrictions refer to clusters that are located in a DMZ, behind a proxy server or a firewall, or that have additional requirements for access.
+
+<p class="message--note"><strong>NOTE: </strong>Kommander does not support attachment of K3s clusters.</p>
+
+For [attaching existing clusters _without_ networking restrictions][attach_without_network_restrictions], the requirements depend on which Kommander 1.X version you are using. Each version of Kommander supports a specific range of Kubernetes versions. You must ensure that the target cluster is running a compatible version.
+
+For example, Kommander 1.3 supports Kubernetes versions between 1.17.0 and 1.19.x. Any cluster you want to attach using Kommander 1.3 must be running a Kubernetes version in that range
+
+For [attaching clusters _with_ networking restrictions][attach_with_network_restrictions], the Kubernetes version of the cluster you want to attach must be greater than or equal to version 1.19.x. For these types of clusters, the Kommander cluster uses a tunnel to access the clusters you plan to attach. Any managed service you want to expose in the Kommander cluster requires the creation of a reverse proxy, or the use of HTTPS_PROXY environment variables that point to the Kommander tunnel proxy server.
+
+When working with clusters that have networking restrictions, you will need to apply some YAML manifests on the existing cluster, so that Kommander can collect a resulting `kubeconfig` file used to establish the tunnel. When you use the Kommander UI, this is handled for you. If you choose to use the manual [CLI attachment process][manual_cli_attachment], you will apply those manifests as part of the larger procedure.
```

Comment:
Do we need `larger` here? I mean, even though it is a more laborious process, should we just say procedure? I don't know the style here.

```suggestion
When working with clusters that have networking restrictions, you will need to apply some YAML manifests on the existing cluster, so that Kommander can collect a resulting `kubeconfig` file used to establish the tunnel. When you use the Kommander UI, this is handled for you. If you choose to use the manual [CLI attachment process][manual_cli_attachment], you will apply those manifests as part of the procedure.
```

Reply:
Resolved as noted in my branch.